cm-piv.c: In function ‘reload_data’:
cm-piv.c:510:7: error: implicit declaration of function ‘gpa_keytable_ensure’ [-Wimplicit-function-declaration]
  510 |       gpa_keytable_ensure (gpa_keytable_get_secret_instance());
      |       ^~~~~~~~~~~~~~~~~~~
cm-piv.c:510:28: error: implicit declaration of function ‘gpa_keytable_get_secret_instance’ [-Wimplicit-function-declaration]
  510 |       gpa_keytable_ensure (gpa_keytable_get_secret_instance());
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
--- a/src/cm-piv.c
+++ b/src/cm-piv.c
@@ -41,6 +41,7 @@
 #include "gtktools.h"
 #include "convert.h"
 #include "gpa-key-details.h"
+#include "keytable.h"
 
 #include "cm-object.h"
 #include "cm-piv.h"
